user/migrating-from-legacy.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Migrating from legacy to container-based infrastructure
+layout: en
+permalink: /user/migrating-from-legacy/
+---
+
+<div id="toc">
+</div>
+
+## What benefits does container-based infrastructure have?
+
+### Builds start in seconds
+
+Rather than wait for builds to start for a long time, your builds now start in less than 10 seconds. Assuming that capacity is available (which is now a lot easier for us to scale), you'll see your builds going from being scheduled to starting in only a few seconds.
+
+### Faster builds
+
+Mileage may vary based on how heavy your builds are on I/O, but most projects should see an improvement in build times. We'd love to hear from your if you don't or if you see slower build times.
+
+### More available resources in a build container
+
+The new containers have 2 dedicated cores available and 4 GB of memory, vs 1.5 cores and 3GB on our legacy infrastructure. CPU resources are now guaranteed. which means less impact by noisy neighbors on the same host machine. Build times throughout the day should be much more consistent on the new container stack.
+
+### Better network capacity, availability and throughput
+
+Our container-based stack is running on EC2, which means much faster network access to most services, especially those hosted on EC2 as well. Access to S3 is now also a ton faster than on our legacy stack.
+
+### Caching available for open source projects
+
+The best news for open source projects is that our build caching is now available for them too. That means faster build speeds by caching dependencies. Make sure to read the docs before trying it out.
+
+For Ruby projects, it's as simple as adding `cache: bundler` to your .travis.yml.
+
+## How can I use it?
+
+Using our new container-based stack only requires one additional line in your .travis.yml:
```

Comment:
maybe add a note that if they see a log line about being on the container based setup then nothing needs to be done?